ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -429,6 +444,28 @@ export const WorkspaceAbout = fp.flow(
                   </div>
                 )}
             </div>
+
+            <div>
+              <h3 style={{ marginBottom: '0.5rem' }}>Billing</h3>
+              <StyledExternalLink
+                data-test-id='workspace-billing-report'
+                href={this.workspaceGcpBillingSpendUrl(
+                  !(
+                    workspace &&
+                    WorkspacePermissionsUtil.isOwner(workspace.accessLevel)
```

Comment:
This extra logic around passing `disabled` through to the link method should not be necessary. Per your changes, if the link is disabled then the `href` prop is ignored anyways.